webapp/channels/src/components/root/root.tsx

```diff
@@ -366,6 +336,12 @@ export default class Root extends React.PureComponent<Props, State> {
         ) {
             this.setRootMeta();
         }
+
+        if (prevState.configLoaded !== this.state.configLoaded && this.state.configLoaded) {
```

Comment:
Nitpicking a bit, but I'd suggest changing this slightly to make it a bit clearer when we want this to happen
```suggestion
        if (this.state.configLoaded && !prevState.configLoaded) {
```

Reply:
done